app/controllers/user/masquerades_controller.rb

```diff
@@ -0,0 +1,14 @@
+class User::MasqueradesController < Devise::MasqueradesController
```

Comment:
Okay `System::Admin` ? I did this is because all the devise controllers are in `user/`, and masquerade is also about user, it's more consistent to me.